internal/native/cgo/edid.c

```diff
@@ -81,32 +81,28 @@ int set_edid(uint8_t *edid, size_t size)
     if (size != 128 && size != 256)
     {
         errno = EINVAL;
-        return -1;
+        return -2;
```

Comment:
Make each error code distinct in the C code so the CGO code can log correct issues.